app/views/admissions_admin/jobs/show.html.haml

```diff
@@ -1,46 +1,148 @@
-- set_title(t('interviews.administer_interviews'))
+= javascript_include_tag 'job_applications/admissions_admin_job_applications', defer: true
 = javascript_include_tag 'applicants/interviews', defer: true
 
-.section
-  .flash.message
-    = T('interviews.possible_to_set_status', date: @admission.admin_priority_deadline)
+- set_title(t('interviews.administer_interviews'))
+
+.flash.message
+  = T('interviews.possible_to_set_priority', date: @admission.admin_priority_deadline)
 
-= link_to admissions_admin_admission_group_path(@job.admission, @job.group) do
-  .samf-button.plain.mb-3
-    = t("common.back")
+.flex-row.flex-align-center
+  = link_to admissions_admin_admission_group_path(@job.admission, @job.group) do
+    .samf-button.plain
+      = t("common.back")
+  = link_to "", { :onclick => "history.go(0)" } do
+    .samf-button.blue.ml-2
+      = t("interviews.refresh_page")
+  %i.ml-2{style: "color: #666"}= t("interviews.refresh_notice")
 
-%h1
-  = t("interviews.administer_interviews")
 
-%h2
+%h1.mt-3
   = t('interviews.job_title', job: @job.title) + " - " + @group.name
 
--#.samf-container.p-3.mt-3
--#  %h2{style:"color: red; text-transform: uppercase;"}= t("interviews.important_title")
--#  %p{style:"color:black; margin: 0;"}
--#    = t("interviews.important_description")
--#  %hr
--#  .flex-row.flex-justify-between
--#    %b{style: "flex-basis: 30%;"}
--#      = t("interviews.important_status_title")
--#    %p{style:"color:black; flex-basis: 70%; margin: 0;"}
--#      = t("interviews.important_status_description")
--#  %hr
--#  .flex-row.flex-justify-between.mb-2
--#    %b{style: "flex-basis: 30%;"}
--#      = t("interviews.important_priority_title")
--#    %p{style:"color:black; flex-basis: 70%; margin: 0;"}
--#      = t("interviews.important_priority_description")
+-# Unprocessed applications
+
+%hr
+
+%h2= t("interviews.unprocessed_applicants") + " (" + @unprocessed_applications.count.to_s + ")"
 
+- if @unprocessed_applications.count > 0
+  %p
+    = t("interviews.unprocessed_explanation")
 
+  = render_async admissions_admin_admission_group_job_show_unprocessed_path(job_id: @job) do
+    .samf-container.flex-col.p-3.text-Align.mb-3
+      %h2= t("interviews.loading_title")
+      %p.mb-2{style: "color: #666;"}= t("interviews.loading_description")
+      .loading-indicator.large.center
 
-= render 'jobs_show', job_application_groupings: [@job.job_applications], show_job_titles: false
+- else
+  %p= t("interviews.no_unprocessed_applicants")
```

Comment:
Would be better if we could show applications in some way, even though they have been processed. Some hidden field that could be expanded, perhaps?